Assets/Scripts/ProjectileSpawner.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ProjectileSpawner : MonoBehaviour
```

Comment:
Cool, this is what I'll use to have a mobile spawner (the ship).

Reply:
Yea, hopefully this is useful